src/routes/Authors.jsx

```diff
@@ -0,0 +1,5 @@
+export default function Authors() {
+  return (
+    <button type="button">Check status</button>
+  );
```

Comment:
- It will be better if we stick to the project requirements as much as possible by changing the `Authors` component to `categories` component. 🤗 

<img width="817" alt="Screenshot 2023-02-24 at 20 25 05" src="https://user-images.githubusercontent.com/99246359/221331477-4f4a76fd-450b-4f06-9c93-43e9a2f87e8c.png">